src/model/ai/AlphaBetaSearchNode.java

```diff
@@ -38,22 +42,18 @@ public AlphaBetaSearchNode(Board board, int depthLevel, PieceColor currentPlayer
         return this.children;
     }
 
-    public Board getBoard() {
-        return this.board;
-    }
-
     public Move getMoveThatGotToThisState() {
         return this.moveThatGotToThisState;
     }
 
-    public boolean isLeaf() {
-        return (this.depthLevel == 0 || this.board.isEndState());
-    }
-
     public double getValue() {
         return this.value;
     }
 
+    public boolean isLeaf() {
+        return (this.depthLevel == 0 || this.board.isEndState());
+    }
+
     public void setValue(double value) {
         this.value = value;
     }
```

Comment:
I suspect this is just a move. I am using the autoformat on save function in Eclipse, and it auto formats and reorders things. I thought that's what we were all using, so that may have been a miscommunication on my part. Remind me later and I will show you guys how to use this. Takes all the guess work out of formatting your code. Just hit `Ctrl` + `s` and it formats and reorders.